### ISSUE TITLE
[Feature request] Remember if Markdown preview was active when opening the app

### ISSUE THREAD

**Issue by joexmdq** [JasonStein/Notepads#524]
**Is your feature request related to a problem? Please describe.**
I don't believe it's really a problem but just a comfort thing

**Describe the solution you'd like**
Be able to remember if the Markdown preview was active or not when you open the app if you have the option to re open the same files you used last time active.

**Describe alternatives you've considered**
The only alternative is activating the preview each time you open the app.

**Additional context**
-

**Comment by JasonStein** [comment]
So first, the code logic to remember that is already there but I never actually use it after next launch. The reason is because, if there is a bug in MarkdownTextBlock, the app will crash on launch immediately but it won't if you don't show the preview. This will unlikely to happen if you never touch the file between two sessions. But if you changed the file in between two sessions with another editor and put some wired stuff in that could potentially bring down the Markdown preview, then it will be a problem.

**Comment by washere** [comment]
I came here to request just this feature. Why?

* Because Markdown  has become the king format and will only grow. Hundreds of Static Gen platforms, numerous apps and many new users each day.

* There are heavy apps for Markdown but for quickly checking an .md files in folders, or doing quick edits etc, we need a light one that launches quickly. Not the big MD apps we might use for long files, MD toolbars shortcuts & other features.

* This is where Notepads comes in as probably the best candidate. I've tested well over a hundred MD editors. In the light category, Notepads can be the best if the Toggle Preview can be set as permanent in settings.

Currently I have set my .md files to open with Furthermark app which has dual column preview. It does not have other features in Notepads, mainly top tabs for multi files.

I think if we have a settings option for .md permanent preview,with a warning (may cause crashes if files change between sessions) then it is worth the risk.

Another advantage Notepads has is that in the category Modern apps for txt editors, it is already the best.

Thanks for the great app.

**Comment by JasonStein** [comment]
@washere I agree with your points here, that is why I added markdown preview in the first place and I do agree modern plain text editor should have markdown preview by default since it is growing fast and widely adopted.

My main focus right now is on the editor and app side, I will shift my attention to markdown preview once I have free time. There are other issues with current markdown preview implementation which you can check here: https://github.com/JasonStein/Notepads/issues/521

**Comment by washere** [comment]
Hi Jason,

Thanks for the explanation. I think you might be like me, a perfectionist. I might be wrong. I am trying to surf the waves nowadays. I was reading about Jeff Bezos methodology, most successful businessman, which is the opposite of mine.

What he does in his corporations is he sets up numerous working groups.
Initially composed of small experts, 3 to 5, tech or whatever related experts, marketing, management etc.
He asks for working prototypes, super fast.
He puts them out fast too, before they are ready.
He closes down most projects and teams, well over 90% within a few months, they are dead projects/ideas.
But the few  that make it are given more funding and people and make him the richest man ever.

SO I am trying to be unlike my old self and sort of surfing it these days.
Also I noticed , perfectionists like me and probably you are actually much more honest that people like Bezos who wing it.
We also link every thing together and want all to be perfect before going further.
For me it is an exciting experience trying to change, I think I had been judgmental about people like Bezos too, I see a point in what they do now.

I don't know if any of his is relevant to you really nor interesting for you, I think not probably, but thought might help someone else reading here. Probably typing it more for myself than anyone else. :)

Anyway I think this issue should not be linked to fixing the flicker till it is perfect, as my old self would say and Bezos would disagree.

Also the default setting for .md preview can be off, deeply buried in settings.
Plus a short warning (Warning: can cause crashes on launch if files change between sessions, etc) next to the toggle button.

Anyway thanks for the great app and do take your time dealing with real life, until you are free, be it a month's time or a year or whenever.
Much appreciated for the great work already, any enhancement would be a bonus.
Best wishes.
